core/src/test/scala/org/apache/spark/deploy/history/EventLogFileWritersSuite.scala

```diff
@@ -66,7 +66,7 @@ abstract class EventLogFileWritersSuite extends SparkFunSuite with LocalSparkCon
     conf.set(EVENT_LOG_DIR, testDir.toString)
 
     // default config
-    buildWriterAndVerify(conf, classOf[SingleEventLogFileWriter])
+    buildWriterAndVerify(conf, classOf[RollingEventLogFilesWriter])
 
     conf.set(EVENT_LOG_ENABLE_ROLLING, true)
```

Comment:
Is it redundant then?
```suggestion
    conf.set(EVENT_LOG_ENABLE_ROLLING, true)
```

Reply:
Or we want to:
```suggestion
    conf.set(EVENT_LOG_ENABLE_ROLLING, false)
    buildWriterAndVerify(conf, classOf[SingleEventLogFileWriter])
```